src/Directives.jl

```diff
@@ -95,11 +99,20 @@ function parsebracket end
 parsebracket(text :: AbstractString) = parsebracket(getdirective(text)...)
 parsebracket{D}(:: Directive{D}, text) = error("Unknown directive: '$D'")
 
+function parsebracket{D}(TD :: Directive{D}, arg, text)
+    if strip(arg) == ""
+        parsebracket(TD, text)
+    else
+        error("parsebracket not defined for directive '$D' with positional argument.")
+    end
+end
+
 function parsebracket(:: directive"docs", text)
+    eval(Expr(:toplevel, Expr(:using, :Lexicon)))
```

Comment:
> Yes, this would be a little DSL for writing parameters. (and rendering).

Sounds fine then.

> I suppose the fail gracefully concern goes away as... if you want to use that syntax you have to ``using Docile``.

Yes, it's unlikely that we could fine some syntax that looks fine without expansion.

Using directives in docstrings needs

```jl
using Docile
addhook(Docile.Docs.directives)
```

so you do need to opt-in to it. There's not much of a way around that.